File: JAVA/Human.java
public class Human {
    // attribute
    private String nik, nama, jenisKelamin;
    
    // constructor
    public Human() {
        this.nik = "";
        this.nama = "";
        this.jenisKelamin = "";
    }

    public Human(String nik, String nama, String jenisKelamin) {
        this.nik = nik;
        this.nama = nama;
        this.jenisKelamin = jenisKelamin;
    }

    // getter and setter
    public String getNik() {
        return this.nik;
    }

    public String getNama() {
        return this.nama;
    }

    public String getJenisKelamin() {
        return this.jenisKelamin;
    }

    public void setNik(String nik) {
        this.nik = nik;
    }

    public void setNama(String nama) {
        this.nama = nama;
    }

    public void setJenisKelamin(String jenisKelamin) {
        this.jenisKelamin = jenisKelamin;
    }

    // method
    public void makan() {
        System.out.println("oke mangan..");
    }
}

File: PHP/Human.php
<?php
# kelas untuk menampung data human dan method makan
class Human {
    // atribut
    private $nik, $nama, $jenisKelamin;

    // konstruktor
    public function __construct() {
        $this->nik = '';
        $this->nama = '';
        $this->jenisKelamin = '';
    }

    // getter and setter
    public function getNik() {
        return $this->nik;
    }

    public function getNama() {
        return $this->nama;
    }

    public function getJenisKelamin() {
        return $this->jenisKelamin;
    }

    public function setNik($nik) {
        $this->nik = $nik;
    }

    public function setNama($nama) {
        $this->nama = $nama;
    }

    public function setJenisKelamin($jenisKelamin) {
        $this->jenisKelamin = $jenisKelamin;
    }

    // method
    public function makan() {
        echo "oke mangan.." . '<br>';
    }
}

?>